### ISSUE TITLE
rosie disco: enhancements

### ISSUE THREAD

**Issue by arjclark** [metomi/rose#1746]
Captured from: https://github.com/metomi/rose/pull/1732#issuecomment-153343806

- [ ] Removal of 1st line of query.
        (I believe this has been discussed before. The HTML form indexing makes this very difficult on existing logic. If it becomes a big deal in the future, we can rip out the logic and try again.)

- [ ] Paging of results - will require work on the server side. Out of scope.

**Comment by matthewrmshin** [comment]
To be honest, I don't think changes to `rosie disco` web client has a lot of priority. The thing with #1732 is a consistency change so it at least looks like Rose Bush, so it does not look unprofessional. The majority of users use `rosie go`.

> Paging of results - will require work on the server side. Out of scope.

Note that this will affect and require work on `rosie go`.

**Comment by arjclark** [comment]
> Note that this will affect and require work on rosie go.

I assume you mean rosie ws? Number of rows is less of a problem for rosie go than it is for the web browsers/

**Comment by matthewrmshin** [comment]
> I assume you mean rosie ws? Number of rows is less of a problem for rosie go than it is for the web browsers.

Both. `rosie.ws` needs to be modified. Subsequently, `rosie go` needs to be modified to accept queries copied from `rosie disco` web.

**Comment by arjclark** [comment]
> Subsequently, rosie go needs to be modified to accept queries copied from rosie disco web.

True, though since I provided that functionality back in 2012 I don't think anyone's used it...
